experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -168,6 +168,7 @@ describe("SharedTreeObject", () => {
 		root.child = newChild;
 		assert.equal(root.child, newChild);
 		root.optional = numberChild.create(newChild);
+		root.optional = numberChild.create(newChild);
```

Comment:
I'm guessing you're checking that the whole eventing apparatus is cleaned up in such a way that one can do it again.
Consider adding a comment explaining that, otherwise someone might think this is a typo.

Reply:
That's actually not what I'm checking here. I'm simply checking that content can be "set to something that it already is" - we didn't have test coverage for these kinds of cases. One of these cases (inserting `[]` into a list) caused a bug in my new code (because the API "early-outed" and didn't round-trip the edit which triggers my event), and it could in theory happen any time the editing code decides to "early" out for a similar reason. But regardless of my particular bug, I thought I'd have each of our object, list and map setter tests do this sort of thing. I'll add a comment to each.